manifests/s/ScreamingFrog/SEOSpider/16.4/ScreamingFrog.SEOSpider.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: ScreamingFrog.SEOSpider
+PackageVersion: "16.4"
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.2.0
```

```suggestion
ManifestVersion: 1.1.0
```
1.2.0 doesn't exist at the moment.

If you're making an update for this package, only PackageVersion, URL, and Hash needs to be changed.